ambari-web/app/models/stack_service.js

```diff
@@ -75,9 +75,27 @@ App.FileSystem = Ember.ObjectProxy.extend({
   content: null,
   services: [],
 
+  //special case for HDFS and OZONE. They can be selected together
+  shouldAllSelectionBeCleared: function () {
+    const dfsNames = this.get('services').mapProperty('serviceName');
+    const coExistingDfs = ['HDFS', 'OZONE'];
+    this.get('services').filterProperty('isSelected').forEach((s) => {
```

Comment:
What this part of the code for? 
"return true" will terminate only forEach callback.

Reply:
thanks for pointing out